azure-kusto-data/azure/kusto/data/request.py

```diff
@@ -240,14 +242,22 @@ class KustoClient(object):
     _mgmt_default_timeout = timedelta(hours=1, seconds=30).seconds
     _query_default_timeout = timedelta(minutes=4, seconds=30).seconds
 
-    def __init__(self, kcsb):
+    def __init__(self, kcsb, max_pool_size=100):

```

Comment:
Let's put max_pool_size as a static member.
This property is for advanced usage, and should not be part of the API.